docs/client/connecting-kn-to-your-cluster.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Connecting the `kn` Client to your cluster"
+linkTitle: "Connecting to your cluster"
+weight: 10
+type: "docs"
+---
+
+The `kn` Client requires a `kubeconfig` file to connect to your cluster.
```

Comment:
I wouldn't  use `kubeconfig`  as a literal as there is typically no file called `kubeconfig` (instead it's a `config` file in the `~/.kube`. What about's just calling it "Kubernetes client configuraton" instead of "`kubeconfig` file". Or at least render kubeconfig in normal font (as I see that Kubernetes over there also uses the terminology "kubeconfig file")